doc/whats_new/v1.3.rst

```diff
@@ -265,9 +265,16 @@ Changelog
 :mod:`sklearn.linear_model`
 ...........................
 
-- |Enhancement| :class:`SGDClassifier`, :class:`SGDRegressor` and
-  :class:`SGDOneClassSVM` now preserve dtype for `numpy.float32`.
-  :pr:`25587` by :user:`Omar Salman <OmarManzoor>`
+- |Enhancement| :class:`linear_model.SGDClassifier`,
+  :class:`linear_model.SGDRegressor` and :class:`linear_model.SGDOneClassSVM`
+  now preserve dtype for `numpy.float32`.
+  :pr:`25587` by :user:`Omar Salman <OmarManzoor>`.
+
+- |API| Deprecates `n_iter` in favor of `max_iter` in
+  :class:`linear_model.BayesianRidge` and :class:`linear_model.ARDRegression`.
+  `n_iter` will be removed in scikit-learn 1.5. This change makes those
+  estimators consistent with the rest of estimators.
```

Comment:
We need to also mention that `n_iter_` was added to `ARDRegression`.

Reply:
I have done so. Please check and see if it's looking good. Is there another particular reason I have missed behind including `n_iter_` attribute in `ARDRegression`

Reply:
I think it could work if it was on it’s own. WDYT @glemaitre ?

Reply:
On it. I will push the change before EOD.